Integrations/integration-ArcSightESM.yml

```diff
@@ -1167,47 +874,62 @@ script:
     - contextPath: ArcSightESM.ActiveList
       description: Active List is a map of active list resource id => active list
         entries
+      type: Unknown
+    - contextPath: ArcSightESM.ActiveList.ListID
+      description: The ActiveList ID
+      type: list
+    - contextPath: ArcSightESM.ActiveList.Entry
+      description: Active List is a map of active list resource id => active list
+      type: Unknown
     description: Get all entries from Active List
   - name: as-get-security-events
-    description: Returns security event object by providing id
     arguments:
-    - name: startDate
-      description: 'Start time filter. Example: 2017-05-22T10:00:00'
-    - name: endDate
-      description: 'End time filter. Example: 2017-05-22T10:00:00'
     - name: ids
-      description: 'ID or multiple ids separated by comma of security events. Event ID is ArcSight is always a number. Example: 13906590'
       required: true
+      description: 'ID or multiple ids separated by comma of security events. Event
+        ID is ArcSight is always a number. Example: 13906590'
+      isArray: true
+    - name: lastDateRange
+      description: 'Query last events. Must be of format ''number date_range_unit''
+        Examples: (2 hours, 4 minutes, 6 month, 1 day, etc.)'
     outputs:
     - contextPath: ArcSightESM.SecurityEvents
```

Comment:
export more Event fields. where are sourceAddress and destinationAddress/attackerAddress?